app/models/katello/repository.rb

```diff
@@ -483,6 +488,39 @@ def import_system_applicability
       end
     end
 
+    def self.import_distributions
+      self.all.each do |repo|
+        repo.import_distribution_data
+      end
+    end
+
+    def import_distribution_data
+      distribution = Katello.pulp_server.extensions.repository.distributions(self.pulp_id)
+      if distribution.any?
+        distribution = distribution[0]
+        self.update_attributes!(:distribution_version => distribution["version"],
+                                :distribution_arch => distribution["arch"],
+                                :distribution_family => distribution["family"],
+                                :distribution_variant => distribution["variant"],
+                                :distribution_bootable => distribution_bootable?(distribution)
+                                )
+      end
+    end
+
+    def distribution_bootable?(distribution = nil)
```

Comment:
also keep in mind that rails iirc will provide an extra method for all boolean database attributes.  For example if you have an attribute called "enabled", you can access it via:

```obj.enabled```
or 
```obj.enabled?```

so you are technically overriding a function rails provides.